sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -1,3 +1,189 @@
-# Azure Messaging EventHub service
+<!-- cspell:words godoc azeventhubs  -->
+# Azure Event Hubs Client Package for C++
 
-C++ SDK for Azure Messaging EventHub.
+[Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) is a big data streaming platform and event ingestion service from Microsoft. For more information about Event Hubs see: [link](https://docs.microsoft.com/azure/event-hubs/event-hubs-about).
+
+Use the client library `github.com/Azure/azure-sdk-for-cpp/sdk//eventhubs` in your application to:
+
+- Send events to an event hub.
+- Consume events from an event hub.
+
+Key links:
+- [Source code][source]
+- [API Reference Documentation][godoc]
+- [Product documentation](https://azure.microsoft.com/services/event-hubs/)
+- [Samples][godoc_examples]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Event Hubs client module for Go with `go get`:
+
+```bash
+vcpkg install azure-messaging-eventhubs-cpp
+```
+
+### Prerequisites
+
+- A C++ Compiler with C++14 support
+- An [Azure subscription](https://azure.microsoft.com/free/)
+- An [Event Hub namespace](https://docs.microsoft.com/azure/event-hubs/).
+- An Event Hub. You can create an event hub in your Event Hubs Namespace using the [Azure Portal](https://docs.microsoft.com/azure/event-hubs/event-hubs-create), or the [Azure CLI](https://docs.microsoft.com/azure/event-hubs/event-hubs-quickstart-cli).
+
+### Authenticate the client
+
+Event Hub clients are created using an Event Hub a credential from the [Azure Identity package][azure_identity_pkg], like [DefaultAzureCredential][default_azure_credential].
```

Comment:
The`#DefaultAzureCredential` anchor link doesn't work for me, it keeps the page at the top.

Reply:
Fixed link (should be defaultazurecredential)